onebone/signal/filter.py

```diff
@@ -2,9 +2,16 @@
 
 - Author: Kyunghwan Kim
 - Contact: kyunghwan.kim@onepredict.com
+'''
+
+'''hampel_filter.
+
+- Author: Sunjin Kim
+- Contact: sunjin.kim@onepredict.com
+
```

Comment:
Author과 Contact를 경환님것과 합쳐서 아래와 같이 작성요청드립니다. (테스트 코드도 마찬가지)
```suggestion
'''

'''(~모듈 설명~)

- Author: Kyunghwan Kim, Sunjin Kim
- Contact: kyunghwan.kim@onepredict.com, sunjin.kim@onepredict.com

```